From d2e0358ece334ceb057307cb5c6dcc08bfddbe66 Mon Sep 17 00:00:00 2001
From: Maxwell G <maxwell@gtmx.me>
Date: Tue, 10 Jun 2025 14:55:26 -0500
Subject: [PATCH 1/2] tests: fix pytest traceback

shutil.get_terminal_size has an optional `fallback` parameter which
pytest tries to pass internally.
---
 src/scancode/cli_test_utils.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/scancode/cli_test_utils.py b/src/scancode/cli_test_utils.py
index e3c50e9041..3eb8e2d01a 100644
--- a/src/scancode/cli_test_utils.py
+++ b/src/scancode/cli_test_utils.py
@@ -110,7 +110,7 @@ def run_scan_click(
 
     if monkeypatch:
         monkeypatch.setattr(click._termui_impl, 'isatty', lambda _: True)
-        monkeypatch.setattr(shutil , 'get_terminal_size', lambda: (80, 43,))
+        monkeypatch.setattr(shutil , 'get_terminal_size', lambda fallback=None: (80, 43,))
 
     if not env:
         env = dict(os.environ)
-- 
2.53.0

